package.json

```diff
@@ -74,7 +74,7 @@
     "ts-jest": "^24.0.0",
     "ts-node": "^8.3.0",
     "tslint": "^5.19.0",
-    "typescript": ">=3.2.1 <3.8.0"
+    "typescript": ">=3.0.0 <3.8.0 >3.7.0-dev.0"
```

Comment:
Why did the lower number change for this? (3.2.1 to 3.0.0)

Reply:
ah shoot. sorry - I was playing around with version ranges to figure out how to include the dev tag, and accidentally dropped the number.
good catch, I'll fix it.